src/main/java/org/mqttbee/api/mqtt/mqtt3/exceptions/Mqtt3MessageException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 The MQTT Bee project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.mqttbee.api.mqtt.mqtt3.exceptions;
+
+import org.mqttbee.annotations.NotNull;
+
+/**
+ * @author David Katz
+ */
+public class Mqtt3MessageException extends Exception {
```

Comment:
This class needs a method "Mqtt3Message getMqttMessage()" similar to the Mqtt5MessageException

Reply:
done, but not currently able to call constructor with Mqtt3Message